src/java.base/share/classes/sun/util/locale/BaseLocale.java

```diff
@@ -33,6 +33,7 @@
 package sun.util.locale;
 
 import jdk.internal.misc.CDS;
+import jdk.internal.util.StaticProperty;
 import jdk.internal.vm.annotation.Stable;
 import sun.security.action.GetPropertyAction;
```

Comment:
unused import can be removed "sun.security.action.GetPropertyAction"

Reply:
Thanks, Sean. Will remove this unnecessary import before the integration.